src/blockchain_state_channel_v1.proto

```diff
@@ -32,29 +44,44 @@ message blockchain_state_channel_v1 {
 message blockchain_state_channel_response_v1 {
     bool accepted = 1;
     packet downlink = 4;
+    bytes active_sc = 5;
 }
 
 message blockchain_state_channel_packet_v1 {
     packet packet = 1;
     bytes hotspot = 2;
     bytes signature = 3;
-    enum Region {
-      US915 = 0;
-      EU868 = 1;
-      EU433 = 2;
-      CN470 = 3;
-      CN779 = 4;
-      AU915 = 5;
-      AS923 = 6;
-      KR920 = 7;
-      IN865 = 8;
-    }
     Region region = 4;
 }
 
+message blockchain_state_channel_offer_v1 {
+    uint32 devaddr = 1;
+    bytes packet_hash = 2;
+    uint64 payload_size = 3;
+    uint32 seqnum = 4;
```

Comment:
is this the `fcnt` (fcount) if so maybe name it fcount for consistency

Reply:
@Vagabond ^

Reply:
sure, why not